designer/client/page-edit.js

```diff
@@ -9,24 +9,29 @@ class PageEdit extends React.Component {
     const form = e.target
     const formData = new window.FormData(form)
     const title = formData.get('title').trim()
-    const newPath = '/'.concat(title.replace(/[^a-zA-Z ]/g, '').replace(' ', '-')).toLowerCase()
+    const newPath = '/'.concat(title.replace(/[^a-zA-Z ]/g, '').replace(/\s+/g, '-')).toLowerCase()
     const section = formData.get('section').trim()
     const pageType = formData.get('page-type').trim()
     const { data, page } = this.props
 
     const copy = clone(data)
-    const pathChanged = newPath !== page.path
+    const titleChanged = title !== page.title
     const pageIndex = data.pages.indexOf(page)
     const copyPage = copy.pages[pageIndex]
 
-    if (pathChanged) {
+    if (titleChanged) {
```

Comment:
do you use titleChanged anywhere else? if not just the condition here rather than the ref